### PR TITLE
Update licensing

### PR DESCRIPTION
**Pull request by pablogs9**
Hello, some points:
- @lukicdarkoo accept the maintainer invitation to this repo
- @lukicdarkoo add yourself to the NOTICE file [like this](https://github.com/micro-ROS/micro_ros_setup/blob/foxy/NOTICE)
- @lukicdarkoo @ralph-lange confirm that you are ok with the new licensing by accepting this PR

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# If Individual's organization is copyright holder of her contributions add the
+# organization's name, optionally also the contributor's name:
+#
+#     Organization's name
+#         Individual's name <submission corporate email address>
+#
+# Please keep the list sorted.
+
+
```
Comment (ralph-lange) [inline NOTICE:24]:
```suggestion
Darko Lukić <lukicdarkoo@gmail.com>
```
Or even add Cyberbotics Ltd. if this work has been conducted in the context of his employment.
// Cc @lukicdarkoo

**Review by ralph-lange — CHANGES_REQUESTED** [review]

**Review by ralph-lange — APPROVED** [review]

**Comment by ralph-lange** [comment]
I just approved, @pablogs9. 

@lukicdarkoo, please find a suggestion how to add yourself to the NOTICE file above in this thread.

File: NOTICE (hunk 1 above)
Comment (lukicdarkoo) [reply NOTICE:24]: Thank you for bringing this up, it is a good point. I have just spoken to @omichel and we think it is a good idea to put `Cyberbotics Ltd.`.

Comment (lukicdarkoo) [inline NOTICE:24]:
```suggestion
Cyberbotics Ltd.
    Darko Lukic <darko.lukic@cyberbotics.com>
```

Comment (pablogs9) [reply NOTICE:24]: You have permission for pushing to this branch, feel free to add it and I'll merge.

**Pull request merged**